scripts/core/make-hoot-josm-java.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
```

Comment:
This script gets run at the end of the Makefile run by `hoot-josm`. It calls Maven to pull down the external josm jar and then build `hoot-josm.jar` from `hoot-josm` Java classes.